### DISCUSSION TITLE
How to read a write a string with gap?

### DISCUSSION THREAD

**Discussion by TomasTrnkaPLC** [FreeOpcUa/python-opcua#1217] [Q&A]
Hi.
I load data from SQL and then back to PLC. If I read or write only a string "XXXXX" everything works well. But if is there a gap "XXX ZZZ" I cannot read or write a string. How to handle that?

**Comment by swamper123** [comment]
This doesn't seem to be an OPC problem. A PLC is a PLC and not OPC.

**Comment by AndreasHeine** [comment]
What is the Error or StackTrace!?
You need to provide more information if you expect an qualified answer otherwise we play the guessing game ^^

We can start with what PLC Type is it?
Show us your node.set_value(???)/node.get_value(???) methods
Does it work with UaExpert?
